include/mapbox/geometry_io.hpp

```diff
@@ -89,9 +89,124 @@ std::ostream& operator<<(std::ostream& os, const geometry_collection<T>& geom)
 
 namespace feature {
 
-std::ostream& operator<<(std::ostream& os, const null_value_t&)
+inline std::ostream& operator<<(std::ostream& os, const null_value_t&)
 {
-    return os << "[]";
+    return os << "null";
+}
+
+void to_stream(mapbox::feature::property_map const&, std::ostream & dest);
+
+void to_stream(std::vector<mapbox::feature::value> const&, std::ostream & dest);
+
+void quote_string(std::string const& in, std::ostream & dest) {
+    dest << "\"";
+    for (char c : in) {
+        if (c == '"' || c == '\\') {
```

Comment:
just `if (c == '"' )` ?

Reply:
We want to escape a backslash as well, otherwise this might be a little odd?